modules/ROOT/pages/using-the-library.adoc

```diff
@@ -0,0 +1,46 @@
+= Using the library
+:description: Learn how to use the Neo4j GraphQL Library with your technology stack.
+
+== Library features
```

Comment:
I'm also not 100% sure this belongs here long term. That being said it see some value in it for now as it's actually quite hard to get a sense of the library features and how it's configured with the current docs layout as most of it buried

Not sure if anyone has any better ideas for this but maybe we leave for now and take another look once we've gone further with this rework of the docs?